src/eip7594/eip7594.c

```diff
@@ -868,21 +915,25 @@ C_KZG_RET verify_cell_kzg_proof_batch(
     // Compute powers of r, and extract KZG proofs out of input bytes
     ////////////////////////////////////////////////////////////////////////////////////////////////
 
-    /*
-     * Derive random factors for the linear combination. The exponents start with 0. That is, they
-     * are r^0, r^1, r^2, r^3, and so on.
-     */
-    ret = compute_r_powers_for_verify_cell_kzg_proof_batch(
```

Comment:
Kinda confused. Why not use the refactored `compute_r_powers_for_verify_cell_kzg_proof_batch()` directly here? It's currently unused.